src/web/ts/screens.tsx

```diff
@@ -35,9 +35,9 @@ export const About = (): VNode<unknown> => <div class={dialogueBox}>
 
   <p>
     If you need help writing a program, I'd recommend checking out the <a href="https://forums.computercraft.cc/"
-      target="_blank" title ="The CC: Tweaked forums">CC: Tweaked</a> or <a href="http://www.computercraft.info/forums2/"
-      title="The ComputerCraft forums" target="_blank">ComputerCraft</a> forums. <a
-      href="http://www.computercraft.info/wiki/Main_Page" target="_blank" title="The CC: Tweaked wiki">The CC: Tweaked
+      target="_blank" title ="The CC: Tweaked forums">CC: Tweaked</a> or <a href="https://discord.gg/H2UyJXe"
```

Comment:
```suggestion
      target="_blank" title ="The CC: Tweaked forums">CC: Tweaked</a> or <a href="https://discord.computercraft.cc"
```

I tend to use this insted, as then it's easier to update the actual invite link.